provision.sh

```diff
@@ -288,14 +315,51 @@ function provision {
         fi
     fi
 
-    printf "Setting LNS address to ${stackAddr}...\n" >&2
-    setLorafwdKeyQuoted "${sshCmd}" "node" "${stackAddr}"
+    pushConfig  "${gatewayAddr}" \
+                "${apiKey}" \
+                "${stackAddr}/api/v3/gcs/gateways/${gatewayID}/cpf/lorad/lorad.json" \
```

Comment:
`cpf` is too generic ("common packet forwarder"); let's make sure we specifically indicate it's `kerlink` here, like `kerlink-cpf`

Also, why are these specific folders?

Reply:
Yeah, that was https://github.com/TheThingsNetwork/lorawan-stack/pull/1747, but I guess we can still change this since we did not make a release.
The folders are different, because these are 2 independent services - `lorad` and `lorafwd`

Reply:
OK fine. Yeah please change there